install/kubernetes/helm/istio/charts/egressgateway/templates/deployment.yaml

```diff
@@ -28,9 +28,8 @@ spec:
           image: "{{ .Values.global.hub }}/proxyv2:{{ .Values.global.tag }}"
           imagePullPolicy: {{ .Values.global.imagePullPolicy }}
           ports:
-            {{- range $key, $val := .Values.deployment.ports }}
-            - containerPort: {{ $val.containerPort }}
-            {{- end }}
+          - containerPort: 80
```

Comment:
We should enable the end user can customize ports, but we can always add the build-in ports here.

Reply:
Roger, I see what your saying above.  Makes good sense to me.  The only downside is folks won't know what the defaults are, and won't be able to override 80/443/31400 if they should be disabled by the operator.

Cheers
-steve

Reply:
this won't work until you define gateway specs for those ports. And if someone decides to expose these services on different set of ports, the whole thing collapses.

Reply:
Thanks @rshriram , let me do some test for this. 

FYI @morvencao